docs/spec_draft.md

```diff
@@ -2234,6 +2235,39 @@ hidden state.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.round_nearest_afz
+
+### Semantics
+
+Performs element-wise round, rounding to nearest integer with ties away from
+zero, on `operand` tensor and produces a `result` tensor.
+
+### Inputs
+
+| Name      | Type                          |
+|-----------|-------------------------------|
+| `operand` | tensor of floating-point type |
+
+### Outputs
+
+| Name     | Type                          |
+|----------|-------------------------------|
+| `result` | tensor of floating-point type |
+
+### Constraints
+
+  * (C1) `opernad` and `result` have the same type.
+
+### Examples
+
+```mlir
+// %operand = [-2.5, 0.5, 2.5]
+%result = "stablehlo.round_nearest_afz"(%operand) : (tensor<3xf32>) -> tensor<3xf32>
```

Comment:
can you add an example where there is no tie and say the rounding is towards zero

Reply:
this + comments on [RoundNearestEvenOp](https://github.com/openxla/stablehlo/pull/552/files#r1023305353)